src/Entities/Player.h

```diff
@@ -682,6 +689,12 @@ class cPlayer :
 
 	AString m_CustomName;
 
+	/** Displayed skin part bit mask */
+	UInt8 m_SkinParts;
```

Comment:
Is it possible/logical to use the enum here?  Or should C++ enum types not be used for flag variables?  (IE, would `eSkinPart m_SkinParts` or `eSkinParts m_SkinParts` be valid/logical?  Or should I switch to `unsigned`?)